app/models/loan_question.rb

```diff
@@ -9,7 +9,6 @@
 #  internal_name         :string
 #  loan_question_set_id  :integer
 #  migration_position    :integer
-#  number                :integer
```

Comment:
I'll run `db:schema:dump`. Perhaps, that's fix this otherwise of course there is another issue